examples/common-switches.mk

```diff
@@ -61,6 +61,7 @@ LOG_OUTPUT          ?= APP
 endif
 LINK_RAW            ?= 0
 MAC_FILTER          ?= 0
+MESSAGE_USE_HEAP    ?= 0
```

Comment:
nit: what about `ALLOC_MESSAGES` ?

Reply:
It seems alloc is too general. Thoughts?